portafly/src/i18n/locales/en/account_settings/personal/personal_details.yml

```diff
@@ -1,5 +1,74 @@
 ---
+# Account settings > Personal > Personal details
+
 page_title: Personal details | Portafly
-page_title_description: Page title of the subsection for Personal details
+page_title_description: Page title of the subsection for Personal details.
+
+# Top area
+
 body_title: Personal details
-body_title_description: Body title of the page for Personal details
+body_title_description: Body title of the page for Personal details.
+
+# Main area
+
+# Tabs
+
+user_data_title: User data
+user_data_description: This is the title of the form containing the user's personal information.
+change_password_title: Change password
+change_password_description: This is the title of the form containing the user's current password field and new password field.
+
+# User data form
+
+username_label: Username
+username_label_description: Label for the field that holds the current username.
+username_value: {{current_username}}
```

Comment:
What's reason behind this string?

Reply:
This is the value the user will enter in the username field. It is a mandatory field.

Reply:
OK but it shouldn't be included in the translations files, since it's not going to be translated. Thins string is not a value we are going to render in the UI.

Reply:
OK, it just doesn't make sense to me to include lines in our code base that aren't going to be used. But I guess I should leave this to you guys anyway.